python/lsst/source/injection/inject_visit.py

```diff
@@ -104,6 +113,13 @@ class VisitInjectTask(BaseInjectTask):
     _DefaultName = "visitInjectTask"
     ConfigClass = VisitInjectConfig
 
+    def run(self, injection_catalogs, input_exposure, psf, photo_calib, wcs):
+        self.log.info("Fitting flux vs. variance in each pixel.")
+        variance_scale = self.get_variance_scale(input_exposure)
+        self.log.info("Variance scale factor: %.3f", variance_scale)
```

Comment:
As with the coadd-version comment above, I'd recommend moving these log messages into the function, hence allowing you to remove them here.